### DISCUSSION TITLE
LSTM Aggregator in GCN

### DISCUSSION THREAD

**Discussion by CodeGeek04** [pyg-team/pytorch_geometric#7118] [General]
I want to add an LSTM Aggregator into my GCN architecture. Number of input features in 384. Can someone please help me do so, preferably by providing relevant code

**Comment by LukeLIN-web** [comment]
Use GCNConv(aggr = LSTM)  ? 
https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/gcn_conv.py#L243

**Reply by rusty1s** [reply]
Yeah, this is the way to go. You should be able to customize the aggregation of GNN layers by using any `Aggregation` module listed [here](https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#aggregation-operators).
